python/src/nanoarrow/visitor.py

```diff
@@ -49,7 +50,7 @@ def to_pylist(obj, schema=None) -> List:
     return ListBuilder.visit(obj, schema)
 
 
-def to_columns(obj, schema=None) -> Tuple[List[str], List[Sequence]]:
+def to_columns(obj, schema=None, handle_nulls=None) -> Tuple[List[str], List[Sequence]]:
     """Convert ``obj`` to a ``list()` of sequences
 
     Converts a stream of struct arrays into its column-wise representation
```

Comment:
For the line below, but can you then clarify when you get a buffer or when a list?